src/js/rx/components/Breadcrumbs.jsx

```diff
@@ -13,6 +27,11 @@ class Breadcrumbs extends React.Component {
 
     if (pathname.match(/\/\d+$/)) {
       crumbs.push(<Link to="/" key="prescriptions">Prescription Refills</Link>);
+
+      if (this.state.prevPath.includes('history')) {
```

Comment:
This stuff should ideally be polyfilled, but I'm switching to use Regex to avoid any unexpected issues